tests/conftest.py

```diff
@@ -51,13 +51,23 @@ def genie_config():
                 "center": "SAGE",
                 "inputSynId": "syn11601335",
                 "stagingSynId": "syn11601337",
+                "errorsSynId": "syn53239079",
                 "release": True,
                 "mutationInCisFilter": "ON",
             },
             "TEST": {
                 "center": "TEST",
                 "inputSynId": "syn11601340",
                 "stagingSynId": "syn11601342",
+                "errorsSynId": "syn53239081",
+                "release": True,
+                "mutationInCisFilter": "ON",
+            },
+            "GOLD": {
+                "center": "GOLD",
+                "inputSynId": "syn52950860",
+                "stagingSynId": "syn52950861",
+                "errorsSynId": "syn53239077",
```

Comment:
I am hesitant to follow our original plan of creating this nested file structure on synapse: `<center_name>/errors/processing/mutation/failed_annotations_report.tsv` vs just placing the failed annotations in `<center_name>/errors/failed_annotations_report.tsv` just because:

1. We don't have many error reports and don't forsee us having many distinct ones - we have the validation errors, and now the mutation annotation errors
2. We'd have to create that nested structure across all centers (would probably have to automate it otherwise that would take some time
3. increased complexity for testing

Thoughts?

Reply:
I agree with this assessment.  Let;'s just try and keep it simple - we can always make the filename more informative